index.ts

```diff
@@ -202,25 +204,19 @@ try {
       ]);
     }
 
-    const filteredSnapshots = snapshots.filter((snapshot: Snapshot) => {
-      return (
-        !packageOutputFilter ||
-        packageOutputFilter.some((filter) => snapshot.package === filter)
-      );
-    });
+    const filteredSnapshots = githubCommentIncludedPackages
+      ? snapshots.filter((snapshot: Snapshot) =>
+          githubCommentIncludedPackages
+            .split(',')
+            .some((filter) => snapshot.package === filter),
+        )
+      : snapshots;
```

Comment:
The main change is we don't need to filter if there isn't a filter. There was a bug before with the logic for the `!packageOutputFilter ||`

Reply:
Yep, sorry about that 🙏